packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -298,22 +298,36 @@ function UserMenu({
             [subMenuActiveContainerStyle]: isCloud,
           })}
         >
-          <MenuItem href={userMenu?.cloud?.userPreferences}>
+          <MenuItem
+            href={userMenu?.cloud?.userPreferences}
+            data-testid="user-menuitem-cloud-user-preferences"
+          >
             User Preferences
           </MenuItem>
-          <MenuItem href={userMenu?.cloud?.invitations}>
-            {openInvitations ? (
+
+          <MenuItem
+            href={userMenu?.cloud?.invitations}
+            data-testid="user-menuitem-cloud-invitations"
+          >
+            {account?.openInvitations ? (
               <span className={subMenuItemStyle}>
-                Invitations <Badge variant="blue">{openInvitations}</Badge>
+                Invitations{' '}
```

Comment:
Not sure I understand this. If the idea is to add whitespace between "Invitations" and the badge, we should use margin instead of a space.

Additionally, is there a reason we can't just conditionally render the badge? We're repeating rendering the "Invitations" text right now.

Reply:
we add a container to control spacing when we have invitations and don't need to worry about styles/spacing when we're not rendering the badge.

Reply:
Is there a difference beyond having an additional element in the DOM? Otherwise, something like this may be best:

```js
<span className={subMenuItemStyle}>
  Invitations

  {account?.openInvitations &&
    <Badge variant="blue">{account.openInvitations}</Badge>
  }
</span>
```

Reply:
we don't need to add justify-content space-between when it's just "invitations"

Reply:
It also shouldn't matter if it's there, and the implementation is cleaner.

Reply:
👍